src/main/drivers/bus_spi.c

```diff
@@ -744,6 +744,19 @@ uint8_t spiGetExtDeviceCount(const extDevice_t *dev)
     return dev->bus->deviceCount;
 }
 
+// Link two segment lists
+// Note that there is no need to unlink segment lists as this is done automatically as they are processed
```

Comment:
Except for BUS_ABORT ... see comment below

Reply:
Ouch. Luckily nothing ever uses that case. I'll change the code to skip forward to the end of the segment list rather than just return. Thanks for this!